bokeh/core/templates.py

```diff
@@ -35,6 +35,8 @@
 
 FILE = _env.get_template("file.html")
 
+ZEPPELIN_FILE = _env.get_template("zeppelin.html")
+
 NOTEBOOK_LOAD = _env.get_template("notebook_load.html")
```

Comment:
There will also need to be supporting documentation changes. Any place in the sphinx docs where things mention "notebook" generically should be made explicitly "jupyter notebook"